e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -444,7 +444,7 @@ func (mts *MetricbeatTestSuite) thereAreEventsInTheIndex() error {
 				"must": []map[string]interface{}{
 					{
 						"match": map[string]interface{}{
-							"event.module": mts.Query.EventModule,
+							"event.dataset": "system.network",
```

Comment:
@jsoriano this is a system metricseat, as I was not able to fetch any module-specific one.

Reply:
Does it mean that we cannot test modules? Or this is a temporary fix?

Reply:
I'm not sure how to retrieve the module specific metrics. The above [JSON response](https://github.com/elastic/e2e-testing/pull/584#issuecomment-762449643) represents a search cURL on the index, and I do not see any event related to the module. Could it be because the 6.8 branch does not understand 7.7 config file? We are fetching 7.7 branch's config because it's the first version including supported_versions files.

As a workaround, we could fetch config files from 6.8 branch on test time, instead of when preparing the test runtime.

Reply:
> Could it be because the 6.8 branch does not understand 7.7 config file?

Do you see activity from modules in logs?

It could happen in some modules that some option has been added in 7.7 that is not available in 6.8, but in general I think that configuration files should be quite similar.

> As a workaround, we could fetch config files from 6.8 branch on test time, instead of when preparing the test runtime.

Yes, I think we should use configuration from 6.8 to test this version.

Reply:
I checked both, 6.8 and 7.7, and they are equals